2-advanced/dubbo-samples-notify/dubbo-samples-notify-consumer/src/main/resources/application.yml

```diff
@@ -0,0 +1,27 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#dubbo:
+#  application:
+#    name: dubbo-samples-notify-consumer
+#  protocol:
+#    name: dubbo
+#    port: -1
+#  registry:
+#    address: zookeeper://${zookeeper.address:127.0.0.1}:2181
+#dubbo:
+#  reference:
+#    org.apache.dubbo.samples.notify.DemoService.sayHello.onreturn: notify.onReturn
```

Comment:
Has this way of configuration proven not to work well?

Reply:
Yes, this doesn't work, I've added README.md and EmbeddedZooKeeper.